### DISCUSSION TITLE
Any way to batch delete multiple URLs?

### DISCUSSION THREAD

**Discussion by dzg** [YOURLS/YOURLS#3766] [Q&A] [answered]
Is there a plugin to enable deleting several URLs at once?

**Comment by dgw** [comment, marked as answer]
Did you [_check_ the Awesome List](https://github.com/YOURLS/awesome#m) before asking this? "[Mass Remove Links](https://github.com/YOURLS/mass-remove-links/)" adds an extra admin page where you can do exactly this, unless you wanted something like check boxes in the main admin table (which I don't believe anyone has done).

**Reply by dzg** [reply]
Ha, I checked, but I searched for "bulk" and "multiple" and "batch" – never thought to search for "mass".

Thanks!
